src/Identity/Specification.Tests/src/Microsoft.AspNetCore.Identity.Specification.Tests.csproj

```diff
@@ -12,6 +12,7 @@
 
   <ItemGroup>
     <Reference Include="Microsoft.AspNetCore.Identity" />
+    <Reference Include="Microsoft.AspNetCore.Testing" />
```

Comment:
Below, there's `<SuppressBaselineReference Include="Microsoft.AspNetCore.Testing" />`. I'm not sure if that needs to be removed.

Reply:
Yea remove it. This should just be a revert of https://github.com/dotnet/aspnetcore/commit/d10a352e8caec866c4107b2fd4fdc5ce49f7a740